gateway/cli/src/main.rs

```diff
@@ -50,7 +52,13 @@ async fn main() -> anyhow::Result<()> {
 
     let cli = Cli::parse();
     let versioned_api = cli.address.join(V1_API_ENDPOINT)?;
-    let create_client = || GatewayRpcClient::new(versioned_api.clone(), cli.rpcpassword.clone());
+    let create_client = || {
+        GatewayRpcClient::new(
+            versioned_api.clone(),
+            cli.rpcpassword.clone(),
+            cli.rpcjwt.clone(),
```

Comment:
~Now needing both is unfortunate, ideally the password would just be an implementation detail of some function/struct providing fetching the session token. What's the plan to evolve this?~

I see, both are optional.